dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/fat/src/com/ibm/ws/jaxws/fat/EnableSchemaValidationTest.java

```diff
@@ -0,0 +1,553 @@
+        LOG.info("testAddedElementServiceNameTrueGlobalSchemaValidationFalse - Response = " + response);
+
+        assertNull("Expected null response from server, but was" + response, response);
+        assertNotNull("Expected Unmarshalling Error: unexpected element exception is not thrown",
+                      server.waitForStringInLog("No child element is expected at this point."));
+    }
+
+    /**
+     * Test the global setting of enableSchemaValidation=false in server.xml, with an additional element <arg0> following the <response> element
+     * which is ignored when schema validation is disabled.
+     *
+     * Server config - servicename-false-global-truee-server.xml
+     * Expected response - Hello, AddedElement
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testAddedElementServiceNameFalseGlobalSchemaValidationTrue() throws Exception {
+        server.reconfigureServer("EnableSchemaValidationTestServer/servicename-false-global-true-server.xml", "CWWKG0017I");
+
+        // Visit the URL:  http://hostName:hostPort/testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=user
+        String response = runTest(server, "testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=AddedElement");
+        // Log response to output.txt
+        LOG.info("testAddedElementServiceNameFalseGlobalSchemaValidationTrue - Response = " + response);
+
+        assertTrue("Expected successful response, but response was " + response,
+                   response.contains("Hello, AddedElement"));
+    }
+
+    // incorrect serviceName tests
+
+    /**
+     * Test the when the incorrect serviceName is given with enableSchemaValidation is set to true server.xml, with an additional element <arg0> following the <response> element
+     * will cause s No child element is expected at this point exception with the default schema validation enabled.
+     *
+     * Server config - incorrect-servicename-server.xml
+     * Expected response - Unmarshalling Error: cvc-complex-type.2.4.a
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testAddedElementIncorrectServiceNameSchemaValidatinFalse() throws Exception {
+        server.reconfigureServer("EnableSchemaValidationTestServer/incorrect-servicename-server.xml", "CWWKG0017I");
+
+        // Visit the URL:  http://hostName:hostPort/testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=user
+        String response = runTest(server, "testHandlerClient/IgnoreUnexpectedElementTestServiceServlet?target=AddedElement");
+        // Log response to output.txt
+        LOG.info("testAddedElementIncorrectServiceNameSchemaValidatinFalse - Response = " + response);
+
+        assertNull("Expected null response from server, but was" + response, response);
+        assertNotNull("Expected Unmarshalling Error: unexpected element exception is not thrown",
+                      server.waitForStringInLog("unexpected element"));
```

Comment:
Test case does not match the test description? Is null response expected? Which Unmarshalling error is expected in this case?